src/irmin-pack/irmin_pack_intf.ml

```diff
@@ -39,20 +39,7 @@ module type Sigs = sig
     ?no_migrate:bool ->
     string ->
     Irmin.config
-  (** Configuration options for stores.
-
-      @param fresh whether an existing store should be overwritten.
-      @param read_only whether read-only mode is enabled for this store.
-      @param lru_size the maximum number of bindings in the lru cache.
-      @param index_log_size the maximum number of bindings in the index cache.
-      @param index_throttle
-        the strategy to use when the index cache is full and an async
-        [Index.merge] in already in progress. [Block_writes] (the default)
-        blocks any new writes until the merge is completed. [Overcommit_memory]
-        does not block but indefinitely expands the in-memory cache.
-      @param indexing_strategy
-        The {{!Indexing_strategy} indexing strategy} of the backend store.
-        Defaults to {!Indexing_strategy.default}. *)
+  (** Configuration options for stores. See {!Irmin_pack.Conf} for more details. *)
```

Comment:
Linking to the `Irmin_pack.Conf` documentation is a trade-off. A negative is that it makes in-editor documentation likely worse since I don't think any editor lets you follow odoc links, but as a positive, it improves the likelihood of documentation being kept up-to-date since it is next to its source.

There is never a guarantee that documentation is kept in-sync with code, but as a demonstration of the positive, the documentation I deleted incorrectly documents the `merge_throttle` flag as `index_throttle`.

Since I generally prefer reading API documentation in a browser, the negative isn't quite as negative to me, but what do others think?